src/controllers/accounts.js

```diff
@@ -14,6 +14,13 @@ exports.getCollection = function * find () {
   this.body = _.invoke(accounts, 'getDataExternal')
 }
 
+exports.getConnectors = function * () {
+  const accounts = yield Account.findAll({
+    where: { connector: { $ne: null } }
+  })
+  this.body = _.invoke(accounts, 'getDataConnector')
```

Comment:
Ok, let's change it at the next convenient opportunity. This makes me wonder what a "data connector" is.